tests/integration/check-variant-message/test_variant_message.py

```diff
@@ -0,0 +1,63 @@
+import sys
+
+import pexpect
+
+from envparse import env
+
+
+def test_check_variant_message(shell, convert2rhel):
```

Comment:
```suggestion
def test_check_variant_message(convert2rhel):
```
Fixture `shell` not used

Reply:
Changed, thanks!